faust/types/app.py

```diff
@@ -189,6 +189,13 @@ def task(self, fun: TaskArg) -> Callable:
     def timer(self, interval: Seconds, on_leader: bool = False) -> Callable:
         ...
 
+    @abc.abstractmethod
+    def crontab(self,
+                cron_format: str,
```

Comment:
@ask for the cron_format `* * * * *` I couldn't figure out a better type than string, thoughts?